### PR TITLE
TL500 chart split

### PR DESCRIPTION
**Pull request by jfilipcz**
Splitted TL500 chart into two separate charts:
- base (operators/deployments/etc)
- course-content (CRs and configs)

### CODE REVIEW

**Comment by eformat** [comment]
hey @jfilipcz  - yup LGMT - just couple merge conflicts to fix up

**Comment by jfilipcz** [comment]
Hey, I've resolved the conflicts, also added updated release workflow

File: tooling/README.md [hunk 1]
```diff
@@ -20,17 +20,26 @@ This content is also made available as [Helm Releases](http://rht-labs.com/enabl
...
+cd ./tl500-base
 helm dep up
-helm upgrade --install tl500 . --namespace tl500 --create-namespace --timeout=15m
+helm upgrade --install tl500-base . --namespace tl500 --create-namespace --timeout=15m
 ```
+2. Install TL500 Course Content
+```bash
+cd ./tl500-course-content
+helm dep up
+helm upgrade --install tl500-course-content . --namespace tl500 --create-namespace --timeout=1>
```
Comment (ckavili) [inline tooling/README.md:33]: is `1>` typo or some sort of notation for timeout? :D

Comment (jfilipcz) [reply tooling/README.md:33]: Ctrl-C Ctrl-V error:)

**Pull request merged**